salsa-2022-tests/tests/compile_fail.rs

```diff
@@ -1,3 +1,4 @@
+#[rustversion::not(nightly)]
```

Comment:
Add this to make `compile_fail' only run when the rust version is not nightly.
Is it ok to not test on the nightly version?